assets/scss/6-components/story-block/_story-block.scss

```diff
@@ -16,11 +16,21 @@
   @include gap;
   display: grid;
   grid-template-columns: 5fr 7fr;
+  grid-auto-rows: min-content;
```

Comment:
Is this rule doing anything? I removed it and the component looked the same, but I could be missing something.

Reply:
When I was testing out these components for the eventual homepage refactor with queso, I recall there was some faux spacing between rows and adding that rule eliminated it. I also don't see it doing anything now so 🤷‍♀ better off only adding it if we know we need it. Will remove.